telemetry/packages/e2e-tests/playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: './tests',
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
```

Comment:
Why don't we want this?

Reply:
Is recommended by Playwright - I think the motivation is to ensure everything is running stably/reproducibility (E2E tests in particular with web are very side effect-y so want to avoid random race condition stuff). Would be tempted to keep it like this unless we need to speed it up later?

Reply:
fair, makes sense